README.md

```diff
@@ -145,32 +143,38 @@ This plugin creates device plugin endpoints based on the configurations given in
 
 ```json
 {
-    "resourceList":
-    [
-        {
-            "resourceName": "sriov_net_A",
-            "rootDevices": ["02:00.0", "02:00.2"],
-            "sriovMode": true,
-            "deviceType": "netdevice"
+    "resourceList": [{
+            "resourceName": "intel_sriov_netdevice",
+            "selectors": {
```

Comment:
we have support code that update the  selectors to be a list of dict. 
so we can do 
            "selectors": [{
                "vendors": ["8086"],
                "devices": ["154c", "10ed"],
                "drivers": ["vfio-pci"]
            }, {
                "vendors": ["15b3"],
                "devices": ["1015"],
                "drivers": ["mlx5_core"]
            }
Can we add this to this PR?

Reply:
Also we have selector per netdevice which can be useful as well.

Reply:
> we have support code that update the selectors to be a list of dict.
> so we can do
> "selectors": [{
> "vendors": ["8086"],
> "devices": ["154c", "10ed"],
> "drivers": ["vfio-pci"]
> }, {
> "vendors": ["15b3"],
> "devices": ["1015"],
> "drivers": ["mlx5_core"]
> }
> Can we add this to this PR?

Lets discuss this use case in out weekly call. A demo/slide would be helpful.

Reply:
> Also we have selector per netdevice which can be useful as well.

I thinking of creating maintaining a new branch until this is merged into master. So that we can include feature improvements, bug tracking/fix etc.